R/reportRenovation.R

```diff
@@ -25,130 +37,151 @@ reportRenovation <- function(gdx, brickSets = NULL, silent = TRUE) {
 
 
 
+  # FUNCTION -------------------------------------------------------------------
+
+  # Replace the pattern if it is a separate vector entry
+  .replaceInVec <- function(pattern, replacement, char) {
+    sub(paste0("^", pattern, "$"), replacement, char)
```

Comment:
I think we don't actually need the regex functionality here and can work with fixed patterns, right?
```suggestion
    char[char == pattern] <- replacement
    char
```